dev/com.ibm.websphere.appserver.features/visibility/public/appClientSupport-1.0/com.ibm.websphere.appserver.appClientSupport-1.0.feature

```diff
@@ -6,6 +6,7 @@ IBM-ShortName: appClientSupport-1.0
 Subsystem-Name: Application Client Support for Server 1.0
 -features=com.ibm.websphere.appclient.appClient-1.0, \
   com.ibm.websphere.appserver.injection-1.0, \
+  com.ibm.websphere.appserver.eeCompatible-6.0; ibm.tolerates:="7.0, 8.0", \
```

Comment:
I had thought about that.  Are you thinking to do it for all features?

Reply:
There is a risk to reordering all features because it may influence the first choice made by the resolver for a solution based on the first preferred version found.  So that could change the features that get enabled for a specific configuration when multiple valid solutions exist.  Here I was really just questioning the addition of the eeCompatible and if we should put that first here.

Reply:
I updated it to be first.  I made sure it was before other tolerate features originally, but it is fine to be first.